core/base/document.php

```diff
@@ -1065,26 +1065,39 @@ public function convert_to_elementor() {
 		}
 
 		// TODO: Better coding to start template for editor
-		return [
+		$internal_elements = [
```

Comment:
Mmm... not sure I understand what `$internal_elements` means? Is there maybe a better descriptive name you can think of?